### PR TITLE
#128481605  Create community  related models

### PR DESCRIPTION
**Pull request by andela-mochieng**
#### What does this PR do?
It creates community-related models.

#### Description of Task to be completed?
Create  and run migrations for community-related models.

#### Any background context you want to provide?
Create a Codango community app that enables users to create and join groups.

#### What are the relevant pivotal tracker stories?
128481605

### CODE REVIEW

File: codango/community/models.py [hunk 1]
```diff
@@ -55,23 +55,26 @@ class Community(TimeStampMixin):
...
     tags = models.ManyToManyField(Tag)
     default_group_permissions = MultiSelectField(
         choices=group_permissions, default='Block members')
 
+    def get_no_of_members(self):
+        return len(self.members.all())
+
     def __str__(self):
-        return self.name
+        return '{} ,{} members'.format(self.name, self.get_no_of_members())
```
Comment (andela-cobiora) [inline codango/community/models.py]: should be `{}, {}`

File: codango/community/migrations/0004_auto_20160830_1041.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('community', '0003_auto_20160824_1103'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='communitymember',
```
Comment (andela-cobiora) [inline codango/community/migrations/0004_auto_20160830_1041.py]: which model is named `communitymember`? @andela-hoyeboade @andela-mochieng

Comment (andela-hoyeboade) [inline codango/community/migrations/0004_auto_20160830_1041.py]: The communitymember model represents the member of a community. It has fields like the username of the member and invitor, the community which the member belong to, the permissions allowed for that member and other fields

File: codango/community/models.py (hunk 1 above)
Comment (andela-hoyeboade) [inline codango/community/models.py]: Fixed

**Pull request merged**